src/rprblender/nodes/blender_nodes.py

```diff
@@ -1148,7 +1148,11 @@ def export(self):
             elif op == 'MODULO':
                 res = self.create_arithmetic(pyrpr.MATERIAL_NODE_OP_MOD, in1, in2)
             else:
-                raise ValueError("Incorrect math operation", op)
+                in3 = self.get_input_value(2)
+                if op == 'MULTIPLY_ADD':
+                    res = in1 * in2 + in3
+                else:
+                    raise ValueError("Incorrect math operation", op)
```

Comment:
With so many new Math and Vector Math options it would be much better to use something like
                log.warn(f"Ignoring unsupported Math operation {op}")
                return None
instead of raising ValueError

Reply:
Agree with such proposition